debian/control

```diff
@@ -1,15 +1,18 @@
 Source: lutris
 Section: games
 Priority: optional
-Build-Depends: cdbs,
- debhelper,
+Build-Depends: debhelper,
+ libsdl2-2.0-0:i386,
+ libstdc++6:i386,
+ libgcc1:i386,
+ libc6:i386,
  python,
  dh-python,
  gir1.2-gtk-3.0,
  gir1.2-glib-2.0,
  python-gi,
- libgirepository1.0-dev,
-Maintainer: Mathieu Comandon <strider@strycore.com>
+ libgirepository1.0-dev
+Maintainer: Michael DeGuzis <mdeguzis@gmail.com>
```

Comment:
I don't think it will build without them. The culprit is koku-xinput-wine, which I am now familiar with, as I have a Debian package for it now. I'll make sure though. That is why they are there. 

On February 24, 2016 6:09:44 PM EST, Mathieu Comandon <notifications@github.com> wrote:
>>   python,
>>   dh-python,
>>   gir1.2-gtk-3.0,
>>   gir1.2-glib-2.0,
>>   python-gi,
>> - libgirepository1.0-dev,
>> -Maintainer: Mathieu Comandon <strider@strycore.com>
>> + libgirepository1.0-dev
>> +Maintainer: Michael DeGuzis <mdeguzis@gmail.com>
>
>yep, can you add another commit with the original maintainer address
>and with the i386 libs removed (OBS doesn't provide multi-arch). Other
>than that it looks cool, I'll test in for next release. 
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/lutris/lutris/pull/256/files#r54023796

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.